### PR TITLE
Change retrieved study output file to use .multipart extension and add Accept headers

### PR DESCRIPTION
**Pull request by noerog**

### CODE REVIEW

File: healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java [hunk 1]
```diff
@@ -65,7 +65,10 @@ public static void dicomWebRetrieveStudy(String dicomStoreName, String studyId)
     // Execute the request and process the results.
     HttpResponse response = request.executeUnparsed();
 
-    String outputPath = "study.dcm";
+    // When specifying the output file, use an extension like ".multipart."
+    // Then, parse the downloaded multipart file to get each individual
+    // DICOM file.
+    String outputPath = "study.multipart";
```
Comment (lesv) [inline healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: I think your comment is probably correct english, but incorrect engineering.  Should it be `".multipart".` instead? L68 or L71 is wrong.

Comment (noerog) [reply healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: You're correct. Changed to ".multipart". to be clearer.

File: healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java [hunk 2]
```diff
@@ -90,11 +93,14 @@ private static CloudHealthcare createClient() throws IOException {
         GoogleCredential.getApplicationDefault(HTTP_TRANSPORT, JSON_FACTORY)
             .createScoped(Collections.singleton(CloudHealthcareScopes.CLOUD_PLATFORM));
 
+    HttpHeaders headers = new HttpHeaders();
+    headers.set("X-GFE-SSL", "yes");
```
Comment (lesv) [inline healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: Probably should explain this.

**Comment by lesv** [comment]
Note - Java 8 is failing.

File: healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java (hunk 2 above)
Comment (noerog) [reply healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: Added an explanation.

**Comment by noerog** [comment]
> Note - Java 8 is failing.

Whoops, typo.

File: healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java (hunk 2 above)
Comment (lesv) [reply healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: where?

Comment (lesv) [reply healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: found it.

Comment (lesv) [reply healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: I was speaking about the X-GFE-SSL header as well.

**Comment by lesv** [comment]
J8 failed last time, just updated, it doesn't appear that you fixed it.

File: healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java (hunk 2 above)
Comment (noerog) [reply healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: That's in all of the samples that dzlier wrote. Should I add it to every sample as well?

**Comment by noerog** [comment]
> J8 failed last time, just updated, it doesn't appear that you fixed it.

Yup, it was actually a new issue introduced. Fixed now.

File: healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java (hunk 2 above)
Comment (lesv) [reply healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: Is a customer likely to see this w/o seeing dziller's code?  If so, you should explain it.  Frankly, I hadn't seen it before.

We can't explain things enough.  The standard I use is, if a user familiar with coding, the language, comes to this code, will they understand it, if not, we should comment.

Comment (noerog) [reply healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: I spoke to dzlier and he said that it's actually not necessary. I'll need to go through all of the other Healthcare samples and remove it wherever it's OK to, but for now I've removed it from this sample.

Comment (lesv) [reply healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java]: Cool

**Pull request merged**